skimage/segmentation/_felzenszwalb.py

```diff
@@ -52,6 +52,10 @@ def felzenszwalb(image, scale=1, sigma=0.8, min_size=20, multichannel=True):
     >>> segments = felzenszwalb(img, scale=3.0, sigma=0.95, min_size=5)
     """
 
+    if not multichannel:
```

Comment:
I think the condition here should be `if not multichannel and image.ndim > 2`, right?